rest-api/test/unit_test/offline_test/biobank_samples_pipeline_mysql_test.py

```diff
@@ -166,9 +166,31 @@ def test_reconciliation_query(self):
     self._insert_samples(p_on_time, BIOBANK_TESTS[:2], ['GoodSample1', 'GoodSample2'], 'OGoodOrder',
                          within_24_hours, within_24_hours - datetime.timedelta(hours=1))
 
+    # On time, recent order and samples not confirmed shows up in missing;
```

Comment:
So these don't show up in missing, nor do the ones below. Please update the comment; and then add some orders which *do* show up in missing, because they use two_days_ago in place of order_time.

And then also add an unconfirmed sample which doesn't have a matching order, with confirmed=two_days_ago, and make sure it *doesn't* show up in missing.

Reply:
done, and as discussed...
confirmed (2 days ago) without order = missing